kuma/wiki/tests/test_views.py

```diff
@@ -417,16 +417,15 @@ def test_basic_view(self, mock_kumascript_get):
         """When kumascript timeout is non-zero, the service should be used"""
         mock_kumascript_get.return_value = (self.doc.html, None)
         self.client.get(self.url, follow=False)
-        ok_(mock_kumascript_get.called, "kumascript should have been used")
+        assert mock_kumascript_get.called, "kumascript should have been used"
 
     @override_config(KUMASCRIPT_TIMEOUT=0.0)
     @mock.patch('kuma.wiki.kumascript.get')
     def test_disabled(self, mock_kumascript_get):
         """When disabled, the kumascript service should not be used"""
         mock_kumascript_get.return_value = (self.doc.html, None)
         self.client.get(self.url, follow=False)
-        ok_(not mock_kumascript_get.called,
-            "kumascript not should have been used")
+        assert not mock_kumascript_get.called, "kumascript not should have been used"
```

Comment:
The original wording of the error message is really awkward. We should improve it with `kumascript should not have been used` instead.